### ISSUE TITLE
Feature Request: Highlight Text Marker

### ISSUE THREAD

**Issue by dCo3lh0** [foobnix/LibreraReader#1051]
Normally Physical book readers wants to have techniques to study better, and one of them is highlighting interesting topics and stuff you find interesting that caught your attention, so if you can select text and highlight text with some predefined colours in the background of that text it will help alot people who only have ebooks and dont wanna print.

Little Example:
https://www.qualtrics.com/support/wp-content/uploads/2017/12/highlight-q-3.png

**Comment by CR0CKER** [comment]
I agree with this and had suggested it in the past. Seems like a missing feature.
